spec/classes/plugin_gutterball_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe 'katello::plugin::gutterball' do
+  let(:foreman) { {:plugin_prefix => 'ruby193-'} }
```

Comment:
Does this work? I think we mostly use pre_condition:
```ruby
let :pre_condition do
  "class {'foreman': plugin_prefix => 'ruby-193-'}"
end
```

Though in this particular case you don't have to specify the plugin_prefix explicitly since it should detect it from the provided facts.